src/main/scala/com/databricks/spark/sql/perf/tpcds/Tables.scala

```diff
@@ -257,6 +276,17 @@ class Tables(sqlContext: SQLContext, dsdgenDir: String, scaleFactor: Int) extend
     }
   }
 
+  def analyzeTables(databaseName: String, analyzeColumns: Boolean = false, tableFilter: String = ""): Unit = {
```

Comment:
minor nit: tableFilter should have been an Option[String], but since it looks like it was a string in `createExternalTables` I guess we can leave it as is ...